libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -40,7 +40,7 @@ public class OrchestratorAdaptiveRecognizer : AdaptiveRecognizer
         /// <summary>
         /// Property key used when storing extracted entities in a custom event within telemetry.
         /// </summary>
-        public const string EntitiesProperty = "entities";
+        public const string EntitiesProperty = "entityResult";
```

Comment:
Should not be named "entities" as there is a RecognizerResult field called "entities" already and when there'll be conflicts when serialized to json,